grafana/dashboards/cmode/power.json

```diff
@@ -92,6 +98,36 @@
         "x": 0,
         "y": 0
       },
+      "id": 99,
+      "panels": [],
+      "title": "Important Information",
+      "type": "row"
+    },
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "gridPos": {
+        "h": 2,
+        "w": 24,
+        "x": 0,
+        "y": 1
+      },
+      "id": 101,
+      "options": {
+        "content": "If an ONTAP system has embedded shelves, it may not have power and temperature information for shelf panels. This is because these details are calculated at the cluster level. Therefore, it is expected that you may not see power and temperature information for shelf panels in such systems.",
```

Comment:
maybe clearer?

If an ONTAP system has **embedded** shelves, you may not see power and temperature information for those shelves since power and temperature are calculated at the cluster level.